includes/RestApi/FlowController.php

```diff
@@ -124,8 +135,10 @@ public function save_onboarding_flow_data( \WP_REST_Request $request ) {
 
 		if ( ( ! empty( $flow_data['data']['siteLogo'] ) ) && ! empty( $flow_data['data']['siteLogo']['id'] ) ) {
 				  \update_option( Options::get_option_name( 'site_icon', false ), $flow_data['data']['siteLogo']['id'] );
+				  \update_option( Options::get_option_name( 'site_logo', false ), $flow_data['data']['siteLogo']['id'] );
```

Comment:
@wpscholar good question, this would be very unlikely as the nesting is defined in the blueprint of our flow data https://github.com/newfold-labs/wp-module-onboarding/blob/ec4ed76ca95381e743ed8b732c3c9d13115e9df8/includes/Data/Flows.php#L51 which means it would always exist from the beginning for 2 reasons

1. If flow data does not exist in the option, the blueprint becomes the new flow data.
2. The only place where flow data is modified is via a REST API in the FlowController which has checks in place to ensure that keys are not deleted/modified/tampered with.
3. If someone were to delete this key directly from the wp_option https://github.com/newfold-labs/wp-module-onboarding/pull/80 has changes that compare the key=>values in the option to the blueprint and recreate any missing keys every time the `load_dashboard_page_*` hook runs.